lib/csl/citation.rb

```diff
@@ -31,43 +17,28 @@ def initialize(pub_hash)
     # @return [Hash]
     def citations
       {
-        apa_citation: to_apa_citation,
-        mla_citation: to_mla_citation,
-        chicago_citation: to_chicago_citation
+        apa_citation: renderer.to_apa_citation,
+        mla_citation: renderer.to_mla_citation,
+        chicago_citation: renderer.to_chicago_citation
```

Comment:
based on the delegation above in line 9, is calling `renderer.` here still necessary?

Reply:
Nope. Removed.